addons/web/static/src/js/views/kanban/kanban_column.js

```diff
@@ -99,6 +99,9 @@ var KanbanColumn = Widget.extend({
         }
         this.$header.find('.o_kanban_header_title').tooltip();
 
+        // Apply a specific class if the column has progressbars
+        this.$el.toggleClass('o_kanban_has_progressbar', self.hasOwnProperty('barOptions'));
```

Comment:
This should be moved below where there is a `if (this.barOptions) {` and use `addClass`

Reply:
Sure, that class is never toggled... just added once. Thanks